oidc-uma-samples/photo-edit/src/main/resources/appone.properties

```diff
@@ -11,7 +11,7 @@ authzEndpoint=https://localhost:9443/oauth2/authorize
 OIDC_LOGOUT_ENDPOINT=https://localhost:9443/oidc/logout
 sessionIFrameEndpoint=https://localhost:9443/oidc/checksession
 tokenEndpoint=https://localhost:9443/oauth2/token
-post_logout_redirect_uri=http://localhost.com:8080/pickup-dispatch/oauth2client
+post_logout_redirect_uri=http://localhost.com:8080/photo-edit/oauth2client
 api_endpoint=http://localhost:39090/bookings
 idp_url=https://localhost:9443
 resource_id=
```

Comment:
Lets add a new line at the end.

Reply:
addressed with c6ecf133e17d127adb2d49c0c9bb5d46664d1941